java/test/jmri/jmrit/display/MemoryIconTest.java

```diff
@@ -91,6 +91,7 @@ protected void init() {
         Assert.assertNotNull("Instance exists", i);
         jmri.InstanceManager.memoryManagerInstance().provideMemory("IM2").setValue("");
         to.setMemory("IM2");
+        to.setMemory("IM2");
```

Comment:
Note that the actual 25d19a6 commit isn't bringing in this change; it's coming in through the previous 7fd2d48 commit which is commented as 
```
Merge branch 'master' of https://github.com/JMRI/JMRI
```
but seems to be carrying non-master changes from its two parents.